spec/takeaway_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'takeaway'
+
+describe TakeAway do
+
+  subject(:takeaway) {described_class.new()}
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments. (https://github.com/bbatsov/ruby-style-guide#no-args-no-parens)